webapp/locales/it.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I guess this solves the problem we had with Italian @Mogge ?
Wonderful !!! 😍

Reply:
?